assgnment2/funcions.dart

```diff
@@ -0,0 +1,183 @@
+
+import 'dart:convert';
+import 'util.dart';
+import 'user.dart';         /// two classes approach (util input, util validation)
+import 'dart:io';
+bool validateUserChoice(int userChoice){
+
+ if( userChoice <1 || userChoice > 5){
+  print('please enter an integer ranging 1-5');
+  return false;
+ }
+ return true;
+
+}
+ int takeUserChoice() {
+
+ int validChoice = Util.validateInteger();// kya ye valid
+
+ while(validateUserChoice(validChoice) == false){
+   validChoice = Util.validateInteger();
+ }
+ return validChoice;
+
+}
+
+void printOptions() {
+
+  print('Enter the as per options below');
```

Comment:
The language of prompts is not right. Make it sound more user-friendly & direct.

For instance, it should simply be `Add a new user` instead of `For adding user details`